README.md

```diff
@@ -207,31 +162,125 @@ Users want to determine several types of the formatting width as following
+  * `days` (`String`): The style to be used for formatting days. Can be one of `"long"`, `"short"`, or `"narrow"`. Defaults to the value of `style`.
+  * `daysDisplay` (`String`): When to display days. `"auto"` means that days won't be displayed if they are zero. Can be either `"always"` or `"auto"`. Defaults to `"auto"` if `days` is `undefined` and `"always"` otherwise.
+  * `hours` (`String`): The style to be used for formatting hours. Can be one of `"long"`, `"short"`, or `"narrow"`. Defaults to the value of `style`.
+  * `hoursDisplay` (`String`): When to display hours. `"auto"` means that hours won't be displayed if they are zero. Can be either `"always"` or `"auto"`. Defaults to `"auto"` if `hours` is `undefined` and `"always"` otherwise.
+  * `minutes` (`String`): The style to be used for formatting minutes. Can be one of `"long"`, `"short"`, or `"narrow"`. Defaults to the value of `style`.
+  * `minutesDisplay` (`String`): When to display minutes. `"auto"` means that minutes won't be displayed if they are zero. Can be either `"always"` or `"auto"`. Defaults to `"auto"` if `minutes` is `undefined` and `"always"` otherwise.
+  * `seconds` (`String`): The style to be used for formatting seconds. Can be one of `"long"`, `"short"`, or `"narrow"`. Defaults to the value of `style`.
+  * `secondsDisplay` (`String`): When to display seconds. `"auto"` means that seconds won't be displayed if they are zero. Can be either `"always"` or `"auto"`. Defaults to `"auto"` if `seconds` is `undefined` and `"always"` otherwise.
+  * `milliseconds` (`String`): The style to be used for formatting milliseconds. Can be one of `"long"`, `"short"`, or `"narrow"`. Defaults to the value of `style`.
+  * `millisecondsDisplay` (`String`): When to display milliseconds. `"auto"` means that milliseconds won't be displayed if they are zero. Can be either `"always"` or `"auto"`. Defaults to `"auto"` if `milliseconds` is `undefined` and `"always"` otherwise.
+  * `microseconds` (`String`): The style to be used for formatting microseconds. Can be one of `"long"`, `"short"`, or `"narrow"`. Defaults to the value of `style`.
+  * `microsecondsDisplay` (`String`): When to display microseconds. `"auto"` means that microseconds won't be displayed if they are zero. Can be either `"always"` or `"auto"`. Defaults to `"auto"` if `microseconds` is `undefined` and `"always"` otherwise.
+  * `nanoseconds` (`String`): The style to be used for formatting nanoseconds. Can be one of `"long"`, `"short"`, or `"narrow"`. Defaults to the value of `style`.
+  * `nanosecondsDisplay` (`String`): When to display nanoseconds. `"auto"` means that nanoseconds won't be displayed if they are zero. Can be either `"always"` or `"auto"`. Defaults to `"auto"` if `nanoseconds` is `undefined` and `"always"` otherwise.
+
```

Comment:
`fractionalDigits` is missing

Reply:
oops, I knew I'd missed something. Adding.